### DISCUSSION TITLE
Support for native sudo

### DISCUSSION THREAD

**Discussion by evaneliasyoung** [Devolutions/UniGetUI#1936] [General]
[Last month, Microsoft introduced the `sudo` command](https://devblogs.microsoft.com/commandline/introducing-sudo-for-windows/) which users can enable if they are using running Windows 11 Insider Dev/Canary version 26052.1100 or higher. Using this `sudo` vs `gsudo` I feel would be a nice improvement.

While this is still an experimental feature, and one in which users are required to enable it themselves, I understand this may not happen for a while. However, I think it noteworthy as a discussion piece.

**Comment by marticliment** [comment]
I have seen it, but I still have to wait until it gets released to beta channel before I can start messing around with it
